### PR TITLE
Sonar Fixes, Part 1

### PR DESCRIPTION
**Pull request by jander99**

### CODE REVIEW

File: local-pipeline.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env sh
```
Comment (emallson) [inline local-pipeline.sh:1]: i like the concept, though this won't work for our windows folks unless they're using WSL i think. i'll probably iterate on this later.

**Review by emallson — APPROVED** [review]
Overall looks good. Can you link the scanner you're using here? If its the kind of static code checker I'm thinking of you're likely to deal with a LOT of false positives from the dataflow tracing.

File: local-pipeline.sh (hunk 1 above)
Comment (jander99) [reply local-pipeline.sh:1]: That's true, I wouldn't know where to begin with a PowerShell equivalent and my .bat file experience is from the 90s 😬

**Comment by jander99** [comment]
> Overall looks good. Can you link the scanner you're using here? If its the kind of static code checker I'm thinking of you're likely to deal with a LOT of false positives from the dataflow tracing.

https://sonarcloud.io/project/overview?id=jander99_WoWAnalyzer

Let me know if you can't access the report, it's their free offering and it's against my fork. Not sure what kind of access I can provide or whether you have to sign up for an account (via Github OAuth).  I haven't tried to understand what it would take to get Github Actions to run a scan yet as it installed a webhook against PRs and the main branch.

**Pull request merged**